iOS Client/ArticlesController/ArticlesCoordinator.swift

```diff
@@ -32,10 +34,28 @@ class ArticlesCoordinator: NSObject, Coordinator, StoreSubscriber, UINavigationC
     
     // MARK: - StoreSubscriber
     func newState(state: MBAppState) {
-        guard state.navigationState.selectedTab == .articles, let newRoute = state.navigationState.routes[.articles] else {
+        guard state.navigationState.selectedTab == .articles,
+            let newRoute = state.navigationState.routes[.articles] else {
             return
         }
         build(newRoute: newRoute)
         route = newRoute
+        
+        let articlesSafariOverlay: URL? = state.navigationState.safariOverlays[.articles]!
```

Comment:
I'm thinking this could get baked into the build method somehow. I'm fine approving for now but I might go back and refactor.

Reply:
cool.

Reply:
yeah feel free